src/npcmove.cpp

```diff
@@ -458,13 +458,18 @@ void npc::execute_action( npc_action action )
 
     case npc_shoot:
         aim();
-        fire_gun( tar, 1 );
+        fire_gun( tar );
```

Comment:
This isn't correct. Before, it would always shoot once (if possible). Now it shoots depending on gun mode.